src/main/resources/config.yml

```diff
@@ -12,7 +12,7 @@ update:
   enabled: true
   strategy: MINOR
   automatic: true
-  ingameNotification: true
+  inGameNotification: true
```

Comment:
Could be undesired due to required user config changes. 
Maybe exclude this change or delay it until the config patcher is done.